fix fetch-incidents returns duplicates bug (#35424)
demisto/content: Packs/CarbonBlackEnterpriseEDR/ReleaseNotes/1_1_35.md

#### Integrations

##### Carbon Black Enterprise EDR

> - Fixed a bug in the **fetch-incidents** which used to return duplicate alerts.

Reviewer: ```suggestion
- Fixed an issue in the **fetch-incidents** which used to return duplicate alerts.
```

> - Fixed an issue in the **fetch-incidents** which used to return duplicate alerts.

Reviewer 2: ```suggestion
Fixed an issue in the **fetch-incidents** which used to return duplicate alerts.
```
